commit 94724c949b6ac8cfba18978d115458d2fa6154a9
Author: Vineetha Hari Pai <vineetha.hari.pai@canonical.com>
Date:   Fri Feb 16 13:41:13 2018 -0500

From: Vineetha Hari Pai<vineetha.hari.pai@canonical.com>
Description: Disable libgcrypt reading /proc/sys/crypto/fips_enabled
file and going into FIPS mode. libgcrypt is not a FIPS
certified library.
Bug-Ubuntu: http://bugs.launchpad.net/bugs/1748310
Forwarded: not-needed

Index: libgcrypt20-1.9.4/src/fips.c
===================================================================
--- libgcrypt20-1.9.4.orig/src/fips.c
+++ libgcrypt20-1.9.4/src/fips.c
@@ -133,6 +133,10 @@ _gcry_initialize_fips_mode (int force)
       goto leave;
     }
 
+  /* Disabling reading fips_enabled file here to prevent
+     the library going automatically into FIPS mode.
+     LP: #1748310 */
+#if 0
   /* Checking based on /proc file properties.  */
   {
     static const char procfname[] = "/proc/sys/crypto/fips_enabled";
@@ -169,6 +173,7 @@ _gcry_initialize_fips_mode (int force)
         abort ();
       }
   }
+#endif
 
   /* Fips not not requested, set flag.  */
   _gcry_no_fips_mode_required = 1;
